gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStoragePathTest.java

```diff
@@ -222,8 +221,8 @@ public void testToRealPath_hasDotDotDir_throwsIae() {
   }
 
   @Test
-  public void testToRealPath_extraSlashes_throwsIae() {
-    try (CloudStorageFileSystem fs = forBucket("doodle")) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/mziccard/gcloud-java/file/2323815760/issues/source?bid=3123107&fileBranchId=3190440#l224)